packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -444,6 +445,34 @@ export const RemoteServerFactory = {
         // This method is used by the dev server, but is not needed here.
     },
 
+    /**
+     * Set up environment variables based on x-forward-* headers
+     * @private
+     */
+    _setXForwardedEnvVars(app, options) {
```

Comment:
There are many headers starting with x-forwarded-* with different purposes (e.g: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) . Using a generic name here makes it easier to add more x-forwarded-* headers in the future.

Reply:
If a client project wanted to use other x-forwarded-* headers, they can take a similar approach as this (by adding an express app in ssr.js), correct?

Is this something we should document?